lib/CfTemplateGenerators/S3.js

```diff
@@ -0,0 +1,18 @@
+function replaceS3BucketFunctionWithAlias(bucket, functionAlias, functionName) {
+  const lambdaConfigurations = bucket.Properties.NotificationConfiguration.LambdaConfigurations;
+  const newConfigurations = lambdaConfigurations.map((configuration) => {
```

Comment:
In this `map` block, you could convert the first two lines into a one liner by using `_.get` as well.